integration-test-example.js

```diff
@@ -0,0 +1,24 @@
+// Integration test file with intentional issues for commenting
+function processUserData(users) {
+    // Missing input validation
+    let results = [];
+
+    // SQL injection vulnerability
+    const query = "SELECT * FROM users WHERE id = " + users[0].id;
+
+    // Hardcoded secrets
+    const apiKey = "sk-1234567890abcdef";
```

Comment:
🔧 API key exposed in code\! What do you think about using environment variables instead? 

```suggestion
const apiKey = process.env.API_KEY;
if (\!apiKey) throw new Error('API_KEY environment variable required');
```